apps/course/src/app/app.component.ts

```diff
@@ -48,3 +48,7 @@ export class AppComponent {
     });
   }
 }
+
+export const Link = {
```

Comment:
Can't this be moved in the `navbar-course.component` itself, since it's only used in the `course-layout` back button and not in the whole app?